### ISSUE TITLE
Snapshot repository not working

### ISSUE THREAD

**Issue by tdjac0bs** [bytedeco/javacpp-presets#462]
Hello, please check if you can help with this error.

I've configured the snapshots repository, as I always do, and it was working in the past, but now it's complaining about missing jar, see:

ERROR:
```
The following artifacts could not be resolved: org.bytedeco.javacpp-presets:opencv:jar:linux-ppc64le:3.2.0-1.3.4-SNAPSHOT, org.bytedeco.javacpp-presets:ffmpeg:jar:linux-ppc64le:3.3.2-1.3.4-SNAPSHOT, org.bytedeco.javacpp-presets:libdc1394:jar:linux-ppc64le:2.2.5-1.3.4-SNAPSHOT, org.bytedeco.javacpp-presets:libfreenect:jar:linux-ppc64le:0.5.3-1.3.4-SNAPSHOT, org.bytedeco.javacpp-presets:artoolkitplus:jar:linux-ppc64le:2.3.1-1.3.4-SNAPSHOT, org.bytedeco.javacpp-presets:flandmark:jar:linux-ppc64le:1.07-1.3.4-SNAPSHOT: Failure to find org.bytedeco.javacpp-presets:opencv:jar:linux-ppc64le:3.2.0-1.3.4-SNAPSHOT in http://oss.sonatype.org/content/repositories/snapshots 
```

The dependency is:
```xml
        <dependency>
            <groupId>org.bytedeco</groupId>
            <artifactId>javacv-platform</artifactId>
            <version>1.3.4-SNAPSHOT</version>
        </dependency>
```

The repository is:
```xml
<repositories>
        <repository>
            <id>oss.sonatype.org-snapshot</id>
            <url>http://oss.sonatype.org/content/repositories/snapshots</url>
            <releases><enabled>false</enabled></releases>
            <snapshots><enabled>true</enabled></snapshots>
        </repository>
    </repositories>
```

**Comment by saudet** [comment]
Set the javacpp.platform system property to the name of your platform.

**Comment by tdjac0bs** [comment]
Thanks, but in this way, the resulting jar will only be compatible with the specified platform, right?

When pointing to release repository i'm able to support all platforms with one build. Is there any way to have it working in the same way in snapshot repository?

**Comment by saudet** [comment]
Well, ideally I'd like to get the linux-ppc64le builds working. @vb216
started to work on that, check in the "ci" subdirectory, if you could help
finish this, that would be great. :)

**Comment by vb216** [comment]
Funny timing, I started looking at this again yesterday, hdf5 was the sticking point but made some progress there, it gets to 80% built now. Rest of the libs cross compile ok (at least, they build, whether they run ok is a different question!)

**Comment by vb216** [comment]
Looking promising - HDF5 just built OK https://travis-ci.org/vb216/javacpp-presets/jobs/273673623 for ppc via cross compile with a few workarounds. Working through the rest.

**Comment by saudet** [comment]
@vb216 How is the progress with the linux-pp64le builds? Anything you're stuck on I might be able to help with?

**Comment by vb216** [comment]
#471 is the PR that will hopefully solve this. It's getting close, just down to ffmpeg, cuda, and leptonica now. ffmpeg and leptonica both struggling to find libraries, cuda needing .so files for ppc to link against. if you're able to move along then great.. works been keeping me pretty busy these last few weeks!

**Comment by saudet** [comment]
Now fixed since pull #471 has been merged. Thanks for the feedback!
